docs/docs/quickstart.md

```diff
@@ -85,51 +61,25 @@ This opens a series of prompts to configure your new project (values in square b
     -   `author_name`: The author of the project
     -   `description`: A short description of your project
     -   `openness`: Whether the project is "public" (default) or "private"
-        -   You should only make a project private if you have a good reason
         -   If "public" then an MIT license will be generated; otherwise the license will be a copyright placeholder
-        -   A Github repo obeying this setting will be created for you in the next section
-    -   `s3_bucket`: The name of an S3 bucket to store assets for this project
-        -   This bucket will be created for you in the next section
-        -   **Careful**: This needs to not conflict with any existing s3 bucket name and [S3 bucket names](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html) must only contain lowercase letters, numbers, dots, and hyphens.
-        -   This value can be reconfigured in `.envrc`.
-    -   `github_account`: The github account that this project will be created in
 
 ### Configure
 
-Run `make init` to perform the following steps:
-
--   `install` - Required by everyone (creator and collaborators) this is itself composed of several sub-steps:
-
-    -   Create a conda environment, `repo_name`, and install the project package in editable mode and its dependencies
-    -   Configure and install Git pre-commit hooks
+When you change directory to your created project folder, you will see that you are in a git repository and the generated cookiecutter has committed itself to the `0_setup_cookiecutter` branch.
 
-        !!! warning inline end "Requires access to Nesta's AWS account"
+Run `make install` to:
 
-    -   Fetch and decrypt the Nesta Metaflow config.
-        It should exist in `~/.metaflowconfig/config_ds-cookiecutter.json`
+-   Create a conda environment (with a name corresponding to the `repo_name` prompt) and install the project package in editable mode and its dependencies
+-   Configure and install Git pre-commit hooks
 
--   Create and configure a github repo `github.com/<github_account>/<repo_name>`
--   Create an S3 bucket `s3_bucket`
+!!! info "Look in [`.recipes/GithubCreation.md`](https://github.com/nestauk/ds-cookiecutter/blob/master/%7B%7B%20cookiecutter.repo_name%20%7D%7D/.recipes/GithubCreation.md) for a quick method of creating and configuring a Github repo for this project."
```

Comment:
It'd be good to have a comment like this pointing us to the `MetaflowAWS.md` document too. Could be here or in faq.md - as I don't think it's referenced atm, but is useful to know it's there.

Reply:
Ah yes I see. I guess as `.recipes/` gets fuller then it will be an overhead to link and explain them all here or in the faq, it just seemed odd at the moment because there are just two and only one gets a mention. I think it's ok to not add to the FAQ either, because you do have a section about customisation. So all good!